src/main/java/ca/nines/ise/writer/XMLWriter.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2014 Michael Joyce <ubermichael@gmail.com>
+ * Copyright (C) 2014 Malcolm Moran <malcolm.moran@outlook.com>
```

Comment:
Sorry, I wasn't sure how I was supposed to handle the copyrights. I think I misunderstood when you told me to add mine; I thought I was supposed to replace yours. It won't happen again.